src/plugins/data/public/query/query_string/index.ts

```diff
@@ -34,6 +34,7 @@ export {
   DatasetService,
   DatasetServiceContract,
   DatasetTypeConfig,
+  DatasetIndexedViewsService as IndexedViewsService,
```

Comment:
The whole point of prefixing is to avoid name collision. Exporting it without a prefix out of here doesn't mitigate that.